VSCODE.md

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
I recommend to rename this file to something along the lines of `devcontainer.md` and link to it from the readme.